### DISCUSSION TITLE
Why next-seo plugin is no longer used?

### DISCUSSION THREAD

**Discussion by Music47ell** [timlrx/tailwind-nextjs-starter-blog#91] [Q&A] [answered]
I noticed that in the v1 branch, specifically in this commit 04ad7617dee5e50ba2e05e58ba76da86e45e2fa4 you decided to replace the [next-seo plugin](https://github.com/garmeeh/next-seo) with a custom header.

May I ask why?
And, are there any plans to bring back JSON-LD support?

**Comment by timlrx** [comment, marked as answer]
I found next-seo to be more complicated than necessary, and that it is easier to add what I want manually rather than through its API. There were also problems with duplicated headers.

I plan to add it back for the blog page. If you are using it for other pages you can generate the code from here https://www.google.com/webmasters/markup-helper/u/0/ and copy it over to the headers.

Hope that helps

**Reply by Music47ell** [reply]
Thank you for your answer. 

Are you currently working on it? Or should I give it a shot and send a PR to the v1 branch?

**Reply by timlrx** [reply]
Feel free to push a PR to the v1 branch, thanks!
